### ISSUE TITLE
racon

### ISSUE THREAD

**Issue by yolanda-huang** [ubccr/software-layer#171]
https://github.com/isovic/racon

**Comment by tonykew** [comment]
Test build of Racon version 1.5.0 complete - more info to follow

@aebruno 
```
tarball:
/srv/cosmos/mirrors/ccr/software-dev/ccr-2023.01-easybuild-tonykew_Racon-1.5.0-GCCcore-11.2.0-2024-01-23.111550.tar.gz 
built with the stock easyconfig file Racon-1.5.0-GCCcore-11.2.0.eb

```

**Comment by tonykew** [comment]
racon version 1.5.0 has been published to the ccrsoft/2023.01 software release

```
vortex1$ module spider racon
----------------------------------------------------------------------------
  racon: racon/1.5.0
----------------------------------------------------------------------------
    Description:
      Ultrafast consensus module for raw de novo genome assembly of long
      uncorrected reads.

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "racon/1.5.0" module is available to load.

      gcccore/11.2.0
[...]
vortex1$ module load gcccore/11.2.0 racon/1.5.0
vortex1$ 
vortex1$ ls ${EBROOTRACON}/bin
racon  racon_preprocess.py  racon_test
vortex1$ 
vortex1$ racon --help
usage: racon [options ...] <sequences> <overlaps> <target sequences>
[...]
vortex1$ 

```
